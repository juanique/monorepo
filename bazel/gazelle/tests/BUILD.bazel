load("@bazel_gazelle//:def.bzl", "gazelle_binary", "gazelle_generation_test")

# gazelle:exclude ./

filegroup(
    name = "test_data",
    srcs = glob(["**/*"]),
)

gazelle_binary(
    name = "gazelle_binary",
    languages = [
        "@bazel_gazelle//language/go",
        "//engprod/gazelle/ts",
    ],
)

gazelle_generation_test(
    name = "gazelle_plugins_test",
    gazelle_binary = ":gazelle_binary",
    test_data = [":test_data"],
)
