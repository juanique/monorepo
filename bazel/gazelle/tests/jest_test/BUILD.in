load("//bazel/js:js.bzl", "jest_test", "js_library")

# This file intentionally empty - Gazelle should generate the appropriate rule
