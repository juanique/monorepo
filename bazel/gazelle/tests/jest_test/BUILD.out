load("//bazel/js:js.bzl", "jest_test", "js_library")

js_library(
    name = "jest_test",
    srcs = ["utils.js"],
)

jest_test(
    name = "utils_test",
    srcs = ["utils.spec.js"],
    deps = [":jest_test"],
)
